db/migrate/20210614145351_create_friendly_id_slugs.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+MIGRATION_CLASS =
+  if ActiveRecord::VERSION::MAJOR >= 5
+    ActiveRecord::Migration["#{ActiveRecord::VERSION::MAJOR}.#{ActiveRecord::VERSION::MINOR}"]
+  else
+    ActiveRecord::Migration
+  end
```

Comment:
We know which version we want, just copy it from another migration instead relying on what friendly brought with it :)